storage/innobase/include/fil0fil.h

```diff
@@ -1373,7 +1364,7 @@ Any other pages were written with uninitialized bytes in FIL_PAGE_TYPE.
 @param[in]	type	expected page type
 @param[in,out]	mtr	mini-transaction */
 #define fil_block_check_type(block, type, mtr)				\
-	fil_page_check_type(block->page.id, block->frame, type, mtr)
+	fil_page_check_type(*block, type, mtr)
```

Comment:
It looks like maybe `fil_page_check_type()` should be renamed to `fil_block_check_type()` and the macro should be removed.

Reply:
Macro removed and PR updated.